book/07-git-tools/sections/revision-selection.asc

```diff
@@ -341,23 +341,23 @@ $ git log ^refA refB
 $ git log refB --not refA
 ----
 
-This is nice because with this syntax you can specify more than two references in your query, which you cannot do with the double-dot syntax.
-For instance, if you want to see all commits that are reachable from `refA` or `refB` but not from `refC`, you can use either of:
+Ito ay maganda dahil kasama ang syntax na ito ay maaari kang magtukoy ng higit dalawang mga reperensiya sa iyong query, na kung saan ikaw ay hindi makagawa gamit ang dobleng-dot na syntax.
+Halimbawa, kung gusto mong makita ang lahat ng mga commit na maaaring mapuntahan mula sa `refA` o `refB` ngunit hindi mula sa `refC`, maaari mong gamitin ang alinman sa:
 
 [source,console]
 ----
 $ git log refA refB ^refC
 $ git log refA refB --not refC
 ----
 
-This makes for a very powerful revision query system that should help you figure out what is in your branches.
+Ginagawa ito para sa isang napaka kapangyarihan na rebisyong query na sistema na dapat makatulong sa iyo na para malaman kung ano ang nasa iyong mga branch.
 
 [[_triple_dot]]
-===== Triple Dot
+===== Triple na Dot
 
-The last major range-selection syntax is the triple-dot syntax, which specifies all the commits that are reachable by _either_ of two references but not by both of them.
-Look back at the example commit history in <<double_dot>>.
-If you want to see what is in `master` or `experiment` but not any common references, you can run:
+Ang huling major na pagitan-pagpili na syntax ay ang tripleng-dot na syntax, na tumutukoy sa lahat ng mga commit na maaaring abutin sa _alinman_ sa dalawang mga reperensiya ngunit hindi sa pamamagitan ng dalawang magkapareho.
```

Comment:
- `major` to `pangunahing`
- you could translate `range-selection` to `pagpili ng hanay`

Reply:
done change